examples/rbfe_edge_list.py

```diff
@@ -26,6 +26,62 @@ def get_mol_by_name(mols, name):
     assert 0, "Mol not found"
 
 
+def run_edge_and_save_results(
+    mol_a,
+    mol_b,
+    core,
+    forcefield,
+    protein,
+    n_frames,
+    seed,
+    smarts,
+    exp_ddg,
+    fep_ddg,
+    fep_ddg_err,
+    ccc_ddg,
+    ccc_ddg_err,
+):
+    mol_a_name = get_mol_name(mol_a)
+    mol_b_name = get_mol_name(mol_b)
+
+    try:
+        complex_res, complex_top = run_complex(mol_a, mol_b, core, forcefield, protein, n_frames, seed)
+        solvent_res, solvent_top = run_solvent(mol_a, mol_b, core, forcefield, protein, n_frames, seed)
+
+        meta = (
+            mol_a,
+            mol_b,
+            smarts,
+            core,
+            float(exp_ddg) * KCAL_TO_KJ,
+            float(fep_ddg) * KCAL_TO_KJ,
+            float(fep_ddg_err) * KCAL_TO_KJ,
+            float(ccc_ddg) * KCAL_TO_KJ,
+            float(ccc_ddg_err) * KCAL_TO_KJ,
+        )
+
+        with open(f"success_rbfe_result_{mol_a_name}_{mol_b_name}.pkl", "wb") as fh:
+            pkl_obj = (meta, solvent_res, solvent_top, complex_res, complex_top)
+            pickle.dump(pkl_obj, fh)
```

Comment:
If the submit goes to a remote host this will write the results to the remote host and not be pulled back to the launcher host.

Reply:
Good point; for now I don't think this is an issue because this code is specialized to `CUDAPoolClient`, which uses local processes. But I was planning to make this abstract over the client as a next step to allow running on remote hosts. Do you think it makes sense to address this then?

Reply:
That's fine too.